cloud-functions/functions/reset-infrastructure.js

```diff
@@ -28,244 +28,140 @@ async function addMinimalSampleData() {
   const initechRef = db.collection('organizations').doc();
   soylentGreenID = soylentGreenRef.id;
   initechID = initechRef.id;
+  await soylentGreenRef.set({
+    name: 'Soylent Green',
+  });
+  await initechRef.set({
+    name: 'Initech',
+  });
+  await createUser({
+    email: 'admin@soylentgreen.com',
+    isAdmin: true,
+    organizationID: soylentGreenID,
+    disabled: false,
+    firstName: uniqueNamesGenerator({ dictionaries: [names], length: 1 }),
+    lastName: uniqueNamesGenerator({ dictionaries: [names], length: 1 }),
```

Comment:
Can we move the `firstName: ... lastName: ...` lines into `createUser()`? They are the same for every invocation.

Reply:
Not worth it imo, this implementation is already built and more flexible.

Reply:
Okay.